.gas-snapshot

```diff
@@ -19,42 +19,48 @@ Certificate_supportsInterface:test() (gas: 5159)
 Certificate_transferFrom:test() (gas: 48618)
 Certificate_transferFrom:test_reverts_when_paused() (gas: 38088)
 Certificate_transferFrom_reverts_ForbiddenTransferAfterMinting:test() (gas: 18960)
-Checkout_buyingFromOneRemoval:test() (gas: 726388)
-Checkout_buyingFromOneRemoval_byApproval:test() (gas: 684637)
-Checkout_buyingFromSingleSupplierWithCustomFee:test() (gas: 547923)
-Checkout_buyingFromTenRemovals:test() (gas: 1976973)
-Checkout_buyingFromTenRemovals_singleSupplier:test() (gas: 1965977)
-Checkout_buyingFromTenRemovals_singleSupplier_byApproval:test() (gas: 1963977)
-Checkout_buyingFromTenRemovals_singleSupplier_withoutFee:test() (gas: 1708460)
-Checkout_buyingFromTenRemovals_withoutFee:test() (gas: 1719464)
-Checkout_buyingFromTenSuppliers:test() (gas: 2855260)
-Checkout_buyingWithAlternateERC20:test() (gas: 544655)
-Checkout_buyingWithAlternateERC20_floatingPointPriceMultiple:test() (gas: 544655)
-Checkout_buyingWithCustomFee:test() (gas: 547446)
-Checkout_swapWithDifferentPermitSignerAndMsgSender:test() (gas: 726838)
+Checkout_buyingFromOneRemoval:test() (gas: 725948)
+Checkout_buyingFromOneRemoval_byApproval:test() (gas: 684373)
```

Comment:
Was the original gas report stale? Is that why we're seeing so many changes here?

If so, can you update the gas report on master first?